src/pages/surveys/[id]/questions/[currentOrder].page.tsx

```diff
@@ -35,8 +40,15 @@ const Question: NextPage = () => {
   ) as SurveyQuestionInterface;
 
   const [isLoading, setIsLoading] = useState(true);
+  const [surveySubmitError, setSurveySubmitError] = useState<string[]>([]);
```

Comment:
```suggestion
  const [surveySubmitErrors, setSurveySubmitErrors] = useState<string[]>([]);
```
😇

Reply:
After reading the whole file, it seems like you do not need an array here... 🤔 
What about using a string and renaming it to "ErrorMessage"? 

```suggestion
  const [submitErrorMessage, setSubmitErrorMessage] = useState('');
```